BookingManager-service/src/main/java/cz/fi/muni/pa165/brown/service/ReservationService.java

```diff
@@ -0,0 +1,59 @@
+package cz.fi.muni.pa165.brown.service;
+
+import cz.muni.fi.pa165.brown.entity.Reservation;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @author Peter Hutta
+ */
+public interface ReservationService {
```

Comment:
Milestone 2 requirements state:

> Make sure that DataAccessException or its subclass is thrown in case of any exception on a the DAO layer

Please consider throwing `cz.muni.fi.pa165.brown.service.exception.ServiceException`